libs/sr/ui/src/LightCone/LightConeCard.tsx

```
@@ -142,15 +142,16 @@ export function LightConeCard({
         >
-          <Typography>
-            HP: {calc.compute(own.base.hp.with('sheet', 'lightCone')).val}
-          </Typography>
-          <Typography>
-            ATK: {calc.compute(own.base.atk.with('sheet', 'lightCone')).val}
-          </Typography>
-          <Typography>
-            DEF: {calc.compute(own.base.def.with('sheet', 'lightCone')).val}
-          </Typography>
+          {(['hp', 'atk', 'def'] as const).map((stat) => (
+            <Typography sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)